config/config.go

```diff
@@ -104,6 +104,8 @@ type Config struct {
 	// RepairMode indicates that the TiDB is in the repair mode for table meta.
 	RepairMode      bool     `toml:"repair-mode" json:"repair-mode"`
 	RepairTableList []string `toml:"repair-table-list" json:"repair-table-list"`
+	// MaxConnections is the maximum permitted number of simultaneous client connections.
+	MaxConnections uint32 `toml:"max-connections" json:"max-connections"`
```

Comment:
I think the config file in MySQL is like TiKV in TiDB for `SET` statement. So we may not need to add a config for it.